worldeditadditions_core/register/handler.lua

```diff
@@ -0,0 +1,40 @@
+function worldeditadditions_core.chatcommand_handler(cmd_name, name, param)
+	local def = assert(worldedit.registered_commands[cmd_name])
+
+	if def.require_pos == 2 then
+		local pos1, pos2 = worldedit.pos1[name], worldedit.pos2[name]
+		if pos1 == nil or pos2 == nil then
+			worldedit.player_notify(name, "no region selected")
+			return
+		end
+	elseif def.require_pos == 1 then
+		local pos1 = worldedit.pos1[name]
+		if pos1 == nil then
+			worldedit.player_notify(name, "no position 1 selected")
+			return
+		end
+	end
+
+	local parsed = {def.parse(param)}
+	local success = table.remove(parsed, 1)
+	if not success then
+		worldedit.player_notify(name, parsed[1] or "invalid usage")
+		return
+	end
+
+	if def.nodes_needed then
+		local count = def.nodes_needed(name, unpack(parsed))
+		safe_region(name, count, function()
```

Comment:
I know this is unfinished, but this is a note to self that safe_region will need defining somewhere. (I'll still merge without this though)

Reply:
This should also go in the issue for `wea.core development`.

Reply:
Indeed